test/apps/iodemo/io_demo.cc

```diff
@@ -1284,6 +1288,7 @@ class DemoServer : public P2pDemoCommon {
             assert(length == opts().iomsg_size);
             validate(conn, msg, length);
         }
+        handle_io_msg(msg, conn);
```

Comment:
maybe move it under `validate` condition? the same for AM and `dispatch_io_message`

Reply:
done, but it requires removing `const` from `pointer to UcxConnection` object

Reply:
sorry for missleading, I ment under `if (opts().validate) {`

Reply:
fixed